cli/ssync/main.go

```diff
@@ -5,13 +5,13 @@ import (
 	"fmt"
 	"os"
 
-	"github.com/rancher/sparse-tools/log"
```

Comment:
Probably it's better to remove the log in an separate commit, convert everything to logrus first.